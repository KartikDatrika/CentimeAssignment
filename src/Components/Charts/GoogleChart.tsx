import { ReactGoogleChartProps } from 'react-google-charts/dist/types';
import Chart from 'react-google-charts';

export default function GoogleChart(props: ReactGoogleChartProps) {
    
    return (
        <div className="mt-5 mb-5">
            <Chart
                {...props}
            ></Chart>
        </div>
    )
}
