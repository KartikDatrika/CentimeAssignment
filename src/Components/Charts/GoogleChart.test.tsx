import React from 'react';
import { render } from '@testing-library/react';
import GoogleChart from './GoogleChart';
import { ReactGoogleChartProps } from 'react-google-charts/dist/types';

describe('Google Chart Test Suite', () => {
  it('Google Chart is rendering without any error !', () => {
    let chartProps: ReactGoogleChartProps = {
      chartType: "Sankey",
      height: 500,
      data:["Inflow", "Outflow", "Amount"]
    }
    const { container } = render(<GoogleChart {...chartProps} />);
    expect(container).toMatchSnapshot();
  });
})
