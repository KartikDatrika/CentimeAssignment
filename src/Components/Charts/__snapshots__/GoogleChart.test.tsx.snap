// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Google Chart Test Suite Google Chart is rendering without any error ! 1`] = `
<div>
  <div
    class="mt-5 mb-5"
  />
</div>
`;
